### DISCUSSION TITLE
SFL Font Letters

### DISCUSSION THREAD

**Discussion by sfwhat** [sunflower-land/sunflower-land#1233] [Art] [answered]
December Update
**Just something for the community use**

I drew letters that are similar to one in our beloved logo, I wanted to share it with community artists for their use on upcoming projects. I could not to make a font file out of this maybe some other contributor can.

### Final Version V4 -- added numbers and symbols

![image (2)](https://user-images.githubusercontent.com/95937808/206933033-64fd91a4-2094-4a45-9e06-74fda408c2e9.png)
![image (1)](https://user-images.githubusercontent.com/95937808/206933034-f1004d3f-3c36-4018-96e5-24d69dec774a.png)

V4 Christmas theme by @BodenSFLpenguin

![image](https://user-images.githubusercontent.com/95937808/206933140-719aa123-2e89-424a-bd48-4ed83c2fb24c.png)
![image (2)](https://user-images.githubusercontent.com/95937808/206933139-6aee6531-8bab-4a31-9121-c2e6d8bcd86b.png)

---

**### V3 Edited one with community suggestions**

![fonvdiscord](https://user-images.githubusercontent.com/95937808/179047377-034e3c02-2a1d-400b-b98a-0ddd7e8b402e.png)
![fonvdiscord-export](https://user-images.githubusercontent.com/95937808/179047383-11162263-44e9-47f0-9541-a72adae970d9.png)

---
**### V2
Chunkier letters**

![chunky](https://user-images.githubusercontent.com/95937808/178928169-d9935c84-5d94-432d-8b1a-ea2f6917a03b.png)

![chunky](https://user-images.githubusercontent.com/95937808/178928211-f4afc498-3451-4cd9-a24c-dd4025e6f2d9.png)

---
**### V1
15x19 even sized letters**

![fontla-large](https://user-images.githubusercontent.com/95937808/178927867-8dec2609-aba4-4d3c-9fd0-be849dfe5ecc.png)

![fontla](https://user-images.githubusercontent.com/95937808/178927870-32f16e28-94ab-4d74-83c6-6fb805a08886.png)

SfWhat
#6127
Twitter @strugwhat

**Comment by digital-farms** [comment]
Why are the letters different thicknesses?🤔

**Reply by sfwhat** [reply]
I got what you mean and edited a new v3

**Comment by adamhannigan** [comment, marked as answer]
Amazing work ❤️ This will be very useful!

**Reply by sfwhat** [reply]
Thank you Adam, glad to help! ❤️

**Reply by irfanshekh9065198863-debug** [reply]
Bast

**Reply by irfanshekh9065198863-debug** [reply]
![Uploading Screenshot_2025-10-19-11-57-30-696_com.android.chrome.jpg…]()

**Comment by stevew00dy** [comment]
I've created a font pack for this - It's only based on PNG and not SVG vectors so it might not work on all programs but I've had success using Photopea online editing software.

https://discord.com/channels/880987707214544966/926888581182652446/1052009371908964362

**Reply by irfanshekh9065198863-debug** [reply]
Plz gift🎁
